tox.ini

```diff
@@ -14,7 +14,7 @@ commands =
     coverage run -m pytest tests -v
     coverage report --show-missing --include=tests/* --fail-under 100
     # This is so that we do not regress unintentionally
-    coverage report --show-missing --include=detect_secrets/* --omit=detect_secrets/core/audit.py,detect_secrets/core/secrets_collection.py,detect_secrets/main.py,detect_secrets/plugins/common/ini_file_parser.py --fail-under 100
+    coverage report --show-missing --include=detect_secrets/* --omit=detect_secrets/core/audit.py,detect_secrets/core/secrets_collection.py,detect_secrets/main.py,detect_secrets/plugins/common/ini_file_parser.py --fail-under 99
```

Comment:
I am not a 💯 fanatic, and okay with regressing coverage. My only issue with not separating out the file(s) is that, now when someone makes a PR, including this one, it's not that clear what isn't covered. `pragma: no coverage` is another option, some people feel it's "lying" (I don't) but it definitely helps with the above.

As an aside, past code like https://github.com/Yelp/detect-secrets/commit/de7fbd265942b1d404a3dc7292db386312a9b63d#diff-895e31d74e6f4c86ab33d77ef2ff05ed  and https://github.com/Yelp/detect-secrets/commit/20d19212de53ae6175cfb2b4aebfd68fe0c18af8#diff-f56910fb7243ad4ed59cd5cf2e1af814 are semi-decent examples of lack of test coverage making things unclear as to why they were written.